src/Surfnet/ServiceProviderDashboard/Application/Dto/EntityDto.php

```diff
@@ -61,16 +35,16 @@ class EntityDto
      * @param string $state
      * @param string $protocol
      */
-    private function __construct($id, $entityId, $environment, $state, $protocol)
-    {
-        $this->id = $id;
-        $this->entityId = $entityId;
-        $this->environment = $environment;
-        $this->state = $state;
-        $this->protocol = $protocol;
+    private function __construct(
+        private $id,
+        private $entityId,
+        private $environment,
+        private $state,
+        private $protocol,
```

Comment:
Did you intentionally not add type definitions here?

Reply:
No, not intentionally. It is just a slevomat multiline rule who did this.